app/konnect/dev-portal/administrators/public-portal.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Public Developer Portal
+no_version: true
+---
+
+Enable public access to your Dev Portal. When Public Portal is enabled, developers
+don't need to register for the portal access and any new application registrations aren't
+allowed. The Register options are not available in the portal.
+
+![Konnect Settings for a Public Portal](/assets/images/docs/konnect/konnect-portal-auto-approve.png)
+
+## Enable a Public Dev Portal
```

Comment:
This reads a bit awkwardly. The goal/action is to make your Dev Portal public, right? So maybe the title should be "Make a Dev Portal public" or something along those lines.

Same for the title below: "Disable public access to a Dev Portal"? As a user, I'd read the title below as disabling a public Dev Portal entirely, ie taking it down, which isn't the case.

Reply:
> This reads a bit awkwardly. The goal/action is to make your Dev Portal public, right? So maybe the title should be "Make a Dev Portal public" or something along those lines.
> 
> Same for the title below: Change "Disable a Public Dev Portal" to "Disable public access to a Dev Portal"? As a user, I'd read the title below as disabling a public Dev Portal entirely, ie taking it down, which isn't the case.

To be consistent with the remainder of the portal docs, we should stick with "Enable" rather than "Make".  So went with "Enable public access to a Dev Portal" and "Disable  Public Access to a Dev Portal" as you suggested. Thanks Lena!